src/System.Net.Sockets/src/System/Net/Sockets/TCPListener.cs

```diff
@@ -93,7 +93,7 @@ public bool ExclusiveAddressUse
         {
             get
             {
-                return _serverSocket.ExclusiveAddressUse;
+                return _serverSocket != null ? _serverSocket.ExclusiveAddressUse : _exclusiveAddressUse;
```

Comment:
In the current version, the user can change this option directly on a socket exposed through Server property and then read it through this property. I just decided to keep this behavior to avoid breaking that implicit contract.